doofinder.php

```diff
@@ -342,12 +360,23 @@ protected function getConfigFormSearchLayer()
     protected function getConfigFormValuesSearchLayer()
     {
         $fields = [];
-
         $fields['DF_INSTALLATION_ID'] = Configuration::get('DF_INSTALLATION_ID');
+        $fields['DF_SHOW_LAYER'] = $this->getShowLayerConfig();
+        $fields['DF_SHOW_LAYER_MOBILE'] = $this->getShowLayerMobileConfig();
```

Comment:
```suggestion
        $fields['DF_SHOW_LAYER'] = Configuration::get('DF_SHOW_LAYER', true);
        $fields['DF_SHOW_LAYER_MOBILE'] = Configuration::get('DF_SHOW_LAYER_MOBILE', true);
```
Why call an intermediate function if you can get the values directly?